protocols/1501-pennington-biomedical/BCA_assay.ot2.py

```diff
@@ -0,0 +1,74 @@
+from opentrons import labware, instruments
+
+metadata = {
+    'protocolName': 'BCA Assay',
+    'author': 'Nick <protocols@opentrons.com>',
+    'source': 'Custom Protocol Request'
+}
+
+# labware
+tube_rack_1 = labware.load('tube-rack-2ml', slot='3')
+tube_rack_2 = labware.load('tube-rack-2ml', slot='6')
+tip_rack_200 = [labware.load('tiprack-200ul', slot) for slot in ['7', '10']]
+trough = labware.load('trough-12row', slot='8')
+plate = labware.load('96-PCR-flat', slot='11')
+
+# pipettes
+p300 = instruments.P300_Single(
+    mount='right',
+    tip_racks=tip_rack_200
+    )
+
+m300 = instruments.P300_Multi(
+    mount='left',
+    tip_racks=tip_rack_200
+    )
+
+
+"""
+create_tube_order outputs the well indices of a 24-well tube rack ordering
+across rows and then down columns
+"""
+
+
+def create_tube_order():
+    new_ind = []
+    for row in range(4):
+        for col in range(6):
+            new_ind.append(col*4 + row)
+    return new_ind
+
+
+def run_custom_protocol(
+        sample_num: int = 32):
+
+    # creates a list of 32 sets of wells in triplicate corresponding to a max
+    # of 32 samples to be transferred
+    well_order = []
+    for move in range(4):
```

Comment:
Use list comprehension verses many for loops.

ie. iterate through the rows, figure out how to split that up into triplicate and then use that list to determine how many wells to fill based on sample number.